cognee/tasks/repo_processor/local_script_dependencies.py

```diff
@@ -0,0 +1,114 @@
+import argparse
```

Comment:
Since this is used for code and I expect more tasks like this, we can put these kind of tasks inside `/cognee/tasks/code` directory.